app/src/main/res/layout/activity_confirm_overtime.xml

```diff
@@ -15,18 +16,272 @@
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:padding="@dimen/dp_10"
         android:orientation="vertical"
         >
-        <Button
-            android:layout_marginTop="@dimen/dp_20"
-            android:layout_width="wrap_content"
+        <android.support.design.widget.AppBarLayout
+            android:id="@+id/appbar"
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_gravity="center_horizontal"
-            android:background="@drawable/border_button_create"
-            android:text="@string/confirm_create"
-            android:textAllCaps="false"
-            android:textColor="@android:color/white"
-            />
+            android:theme="@style/AppTheme.AppBarOverlay"
+            >
+            <android.support.v7.widget.Toolbar
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                app:popupTheme="@style/AppTheme.PopupOverlay"
+                />
+        </android.support.design.widget.AppBarLayout>
+        <ScrollView
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:scrollbars="none"
+            >
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
```

Comment:
WARNING: [ScrollViewSize] This LinearLayout should use `android:layout_height="wrap_content"`
 ScrollView children must set their `layout_width` or `layout_height` attributes to `wrap_content` rather than `fill_parent` or `match_parent` in the scrolling dimension